tests/validators.py

```diff
@@ -378,7 +381,6 @@ def validate_linkerd():
         timeout_insec=300,
     )
     print("Linkerd proxy injector up and running.")
-    ### Disabling this test because the deletion of the namespace get stuck.
```

Comment:
It looks like this test is not disabled, so just removed the comment instead of adjusting formatting.